etc/spotbugs-exclude.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright Kroxylicious Authors.
+  ~
+  ~ Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+  -->
+
+<FindBugsFilter>
+    <!--
+        The kroxylicious-kubernetes-api package-info.java uses @DefaultAnnotation(NonNull.class) like the other modules
+        but default JSON values are usually specified using @io.fabric8.generator.annotation.Default;
+        Null values are used to exclude properties from the final JSON e.g. in combination with
+        @com.fasterxml.jackson.annotation.JsonInclude(com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL)
+        -->
+    <Match>
+        <Source name="~.*io/kroxylicious/kubernetes/api/common/.*"/>
+        <Bug pattern="NP_NONNULL_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR" />
+    </Match>
+    <!-- These are excluded because the classes are generated by sundrio / fabric8 -->
+    <Match>
+        <Class name="~io\.kroxylicious\.kubernetes\.api\.common\..+(Builder|Fluent)(\$.*)?" />
+    </Match>
+    <Match>
+        <Class name="~io\.kroxylicious\.proxy\.config\..+Fluent(\$.*)?" />
+    </Match>
+    <!--
+        Both of the sources below contain classes used for benchmarks.
+        To accurately measure performance, it is useful to sometimes forcibly return null.
+        -->
+    <Match>
+        <Source name="~.*io/kroxylicious/microbenchmarks/.*"/>
+        <Bug pattern="NP_NONNULL_RETURN_VIOLATION" />
+    </Match>
+    <Match>
+        <Source name="~.*io/kroxylicious/filters/.*"/>
```

Comment:
`*io/kroxylicious/filters/` worries me because is could match non-mircobenchmark code.

I wonder if moving OneInterfaceFilter  and friends to io.kroxylicious.microbenchmarks.filters would be a better solution.